lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -367,9 +366,9 @@ <h4 class="message-title">${_('Your verification will expire soon!')}</h4>
                 '<a id="unregister_block_course" rel="leanModal" '
                 'data-course-id="{course_id}" data-course-number="{course_number}" data-course-name="{course_name}" '
                 'href="#unenroll-modal">'.format(
-                  course_id=escape(course_overview.id),
-                  course_number=escape(course_overview.number),
-                  course_name=escape(course_overview.display_name_with_default_escaped),
+                  course_id=course_overview.id,
+                  course_number=course_overview.number,
+                  course_name=course_overview.display_name_with_default,
                 )
               ),
               unenroll_link_end="</a>",
```

Comment:
Is default escaping the right thing to do here if we're trying to render HTML tags?